### PR TITLE
Fixed https://wso2.org/jira/browse/PC-201, https://wso2.org/jira/brow…

### PR DESCRIPTION
**Pull request by dewmini**
…se/PC-202 and the issue in displaying dates in datepicker

### CODE REVIEW

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js [hunk 1]
```diff
@@ -146,6 +147,32 @@ function loadList(dropdownElementID) {
...
+                    el.value = opt;
+                    $(dropdownElementID).append(el);
+                }
+                $(dropdownElementID).selectpicker("refresh");
+            }
+            loadUserList("UserList");
+        },
+        error: function (xhr, status, error) {
+            var errorJson = eval("(" + xhr.responseText + ")");
+            alert(errorJson.message);
```
Comment (prasa7) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: Remove alert message and use a notification plugin.

File: modules/extensions/package/publisher/package/themes/default/partials/list-assets.hbs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+<div class="row">
+    <div class="col-lg-10 col-md-10 col-sm-10 col-xs-9 wr-filters">
+        {{#if assetCategoryDetails.hasCategories}}
+            <div class="wr-filter-category">
+                <a href="#" class="ico-filter ctrl-filter-category dropdown-toggle" data-toggle="dropdown"
+                   aria-expanded="true" id="categoryDropDown" title="All Categories">
+                    {{#if assetCategoryDetails.selectedCategory}}
+                        {{assetCategoryDetails.selectedCategory}}
+                    {{else}}
+                        <span>All Categories</span>
```
Comment (prasa7) [inline modules/extensions/package/publisher/package/themes/default/partials/list-assets.hbs]: Add multi language support to this. Check the whole PR and correct them.

Comment (dewmini) [inline modules/extensions/package/publisher/package/themes/default/partials/list-assets.hbs]: This does not support internationalization.

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js (hunk 1 above)
Comment (dewmini) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js]: This does not support any notification plugin hence keeping the alert.

**Pull request merged**